orc8r/gateway/python/magma/magmad/upgrade/upgrader.py

```diff
@@ -81,26 +83,26 @@ def start_upgrade_loop(magmad_service, upgrader):
 
     while True:
         logging.info('Checking for upgrade...')
-        try:
-            target_ver = _get_target_version(magmad_service.mconfig)
-            upgrader.perform_upgrade_if_necessary(target_ver)
-        except Exception:  # pylint: disable=broad-except
-            logging.exception(
-                'Error encountered while upgrading, will try again after delay',
-            )
+        target_ver = _get_target_version(magmad_service.mconfig)
         poll_interval = max(  # No faster than 1/minute
             60,
             magmad_service.mconfig.autoupgrade_poll_interval,
         )
+        if target_ver != DEFAULT_VERSION:
```

Comment:
This is mostly my preference but I think it might be clearly if it's like
```python
if target_ver == DEFAUL...:
  # log warning
else: 
  # do work
```